### ISSUE TITLE
Soundboard

### ISSUE THREAD

**Issue by dm-helper** [MMetze/DMHelper-Tracker#95]
**Discord User:**
dm-helper

1. Create a soundboard for individual sound effects and sound playback
2. Integrate into the Syrinscape online player for remote sound playback
3. Include Youtube audio playback

**Comment by dm-helper** [comment]
Originally communicated as a part of v1.7, will be postponed past the v2.0 timeline to focus for now on the UI updates.

**Comment by DrkNinja** [comment]
A follow up for this I think could be to make it as simple as there is one greater over all track (which you would set up under audio as like a "general" track and be able to select via dropdown in the soundboard) and then have like a minor track that plays on top of the overall track when a map is published that has a song attached to it. That way you can have like general peaceful travel music going on, and then when you go to a blacksmith have black smith sounds play on top of that travel music.

**Comment by dm-helper** [comment]
Yeah, this is going to be a big one. I think that sort of thing should be configurable. It will also depend on the source, for example Syrinscape and other such systems handle all the mixing of effects, but wouldn't allow multiple tunes at the same time. If you're playing back local or youtube sources, they would have to be mixed locally.
